Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1094,8 +1094,13 @@ extension BrowserViewController: BrowserDelegate {
         browser.addHelper(favicons, name: FaviconManager.name())
 
         // Temporarily disable password support until the new code lands
-        let logins = LoginsHelper(browser: browser, profile: profile)
-        browser.addHelper(logins, name: LoginsHelper.name())
+        // don't understand ^^. Is this still relevant?
```

Comment:
I think this was from when we had that issues with the way we were saving passwords a few months back. I think we can ignore this comment for now.